website/project/model.py

```diff
@@ -1638,7 +1638,7 @@ def use_as_template(self, auth, changes=None, top_level=True):
         new.nodes = [
             x.use_as_template(auth, changes, top_level=False)
             for x in self.nodes
-            if x.can_view(auth)
+            if x.can_view(auth) and not x.is_deleted
```

Comment:
I don't *think* so, but I can see the argument for it. I think there are likely a lot of code assumptions around `can_view` being permissions-only that changing that might cause trouble. Philosophically, probably it should.